playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -129,44 +129,50 @@ def graph_enricher(alert: PrometheusKubernetesAlert, params: PrometheusParams):
     """
     Enrich the alert with a graph of the Prometheus query which triggered the alert.
     """
-    if params.prometheus_url:
-        prometheus_base_url = params.prometheus_url
-    else:
-        prometheus_base_url = PrometheusDiscovery.find_prometheus_url()
-    prom = PrometheusConnect(url=prometheus_base_url, disable_ssl=True)
     promql_query = alert.get_prometheus_query()
-
-    end_time = datetime.now(tz=alert.alert.startsAt.tzinfo)
-    alert_duration = end_time - alert.alert.startsAt
-    graph_duration = max(alert_duration, timedelta(minutes=60))
-    start_time = end_time - graph_duration
-    increment = graph_duration.total_seconds() / 60
-    result = prom.custom_query_range(
+    chart = playbooks_utils.create_chart_from_prometheus_query(
+        params.prometheus_url,
         promql_query,
-        start_time,
-        end_time,
-        increment,
-        {"timeout": PROMETHEUS_REQUEST_TIMEOUT_SECONDS},
+        alert.alert.startsAt,
+        include_x_axis=False,
```

Comment:
include_x_axis means should the y axis always start from 0. it kind of gives perspective, for example if the memory fluctuage between 40 to 45 percent it might look very scary if the y axis starts from 40 instead of 0.